Lib/ast.py

```diff
@@ -620,6 +620,15 @@ class AugStore(expr_context):
 class Param(expr_context):
     """Deprecated AST node class.  Unused in Python 3."""
 
+_Unparser = None
+
+def unparse(ast_obj):
+    global _Unparser
+    if _Unparser is None:
+        from _ast_unparse import Unparser as _Unparser
+    unparser = _Unparser()
+    return unparser.visit(ast_obj)
+
 
 def main():
     import argparse
```

Comment:
We may as well defer the `sys` import? Instructions not executed are always faster!
```suggestion
    import argparse
    import sys
```